debian-8/init_container.sh

```diff
@@ -19,51 +19,17 @@ cat /etc/motd
 sed -i "s/SSH_PORT/$SSH_PORT/g" /etc/ssh/sshd_config
 /usr/sbin/sshd
 
-[ -z "$ASPNETCORE_URLS" ] && export ASPNETCORE_URLS=http://*:"$PORT"
-
-# If there is any command line argument specified, do basic validation and execute.
-if [ $# -ne 0 ]; then
-    echo "App Command Line configured: $@"
-    if [ $# -eq 1 ]; then
-        # ensure $1 exists on disk
-        if [ -f $1 ]; then
-            echo "App Command Line file on disk, executing it."
-            exec "$@"
-        else
-            echo "App command Line file doesn't exist on disk, fallback to auto-detect"
-        fi
-    elif [ $# -eq 2 -a $1 == "dotnet" ]; then
-        if [ -f $2 ]; then
-            echo "Argument to dotnet exists on disk, launching it: $@"
-       	    exec "$@"
-        else
-            echo "Argument to dotnet doesn't exist on disk, fallback to auto-detect"
-        fi
-    else
-        exec "$@"
-    fi
-fi
-
-# Pick up one .csproj file from repository where git push puts files into
-CSPROJ=`ls -1 /home/site/repository/*.csproj 2>/dev/null | head -1`
-
-# Convert /home/site/repository/<name>.csproj into ./<name>.dll and execute it
-if [ -n "$CSPROJ" ]; then
-    DLL=./`basename "$CSPROJ" .csproj`.dll
-    if [ -e $DLL ]; then
-        echo Found: $DLL
-        exec dotnet "$DLL"
-    else
-        if [ -d /home/site/wwwroot/oryx_publish_output ]; then
-            DLL=./oryx_publish_output/`basename "$CSPROJ" .csproj`.dll
-            if [ -e $DLL ]; then
-                echo Found: $DLL
-                exec dotnet "$DLL"
-            fi
-        fi
-    fi
+appPath="/home/site/wwwroot/"
+sourcePath="/home/site/repository/"
+startupCommandPath="/opt/startup/startup.sh"
+defaultAppPath="/defaulthome/hostingstart/hostingstart.dll"
+oryxArgs="-appPath $appPath -sourcePath $sourcePath -output $startupCommandPath -defaultAppFilePath $defaultAppPath \
```

Comment:
With my check-in yesterday into Oryx, `sourcePath` is no longer required or available as switch...probably this needs to go into a different branch?

Reply:
yes I know, but that is not in prod yet.